v3/plugins/ms-vscode/protobuf/0.4.2/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: ms-vscode
+name: protobuf
+version: 0.4.2
+type: VS Code extension
+displayName: Protobuf
+title: Google's protobuf .proto file support
+description: This extension adds rich language support for the proto3 syntax
+icon: https://github.com/zxh0/vscode-proto3/raw/master/images/vscode_extension_icon.png
+repository: https://github.com/zxh0/vscode-proto3
+category: Language
+firstPublicationDate: '2020-08-26'
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-protobuf:0.4.2-a79a2ff"
+      name: vscode-proto3
+      memoryLimit: '256Mi'
+  extensions:
+    - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/zxh404/vsextensions/vscode-proto3/0.4.2/vspackage
```

Comment:
Unfortunately we can't link to the visualstudio marketplace, and it seems the upstream repository doesn't host any built vsix files. I will build this extension and host it on download.jboss.org, for now.

Reply:
I think it's not a problem to build the binary and publish it to https://download.jboss.org/jbosstools/vscode/3rdparty as we did to other extensions.